app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -485,6 +485,10 @@ public boolean setViewValue(View view, Cursor cursor, int columnIndex) {
                         }
                         return true;
                     case R.id.nameTV:
+                        //currently emojis are in png format, add others if new format(jpg, jpeg etc) is introduced
+                        if (!personTable && name.endsWith(".png")) {
```

Comment:
And while saving Emoji string in the database we have to remove ".png"

and for getting Drawable of emoji we have to give full reference of emoji name with ".png"

@kunall17